.github/workflows/build.yaml

```
@@ -91,7 +91,7 @@ jobs:
   web-app:
     needs:
       - lib
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Ugh. I'm guessing we are waiting for playwright? Maybe add a note here?
```suggestion
    runs-on: ubuntu-22.04 # NOTE: Playwright not yet supported on 24.04
```